### PR TITLE
First version of Russian translation

### PR DESCRIPTION
**Pull request by unhappytroll**
First version of Russian translation, for testing purposes, will be corrected in the future

### CODE REVIEW

File: addons/common/stringtable.xml [hunk 1]
```diff
@@ -68,105 +68,139 @@
...
             <English>Aircraft</English>
+            <Russian>Самолёт</Russian>
         </Key>
         <Key ID="STR_ZEN_Common_Filter">
             <English>Filter</English>
+            <Russian>Фильтр</Russian>
         </Key>
         <Key ID="STR_ZEN_Common_Units">
             <English>Units</English>
+            <Russian>Солдаты</Russian>
```
Comment (CreepPork) [inline addons/common/stringtable.xml]: It's not only soldiers, could be just some unit (inc. civilian side)

Comment (unhappytroll) [reply addons/common/stringtable.xml]: that depends on particular point of application. we have quite a bunch of synonyms, which can be applied under that word.

File: addons/attributes/stringtable.xml [hunk 2]
```diff
@@ -1,18 +1,21 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (mharis001) [inline addons/attributes/stringtable.xml]: What changed in these lines?

File: addons/editor/stringtable.xml [hunk 3]
```diff
@@ -1,58 +1,69 @@
...
         </Key>
         <Key ID="STR_ZEN_Editor_DeclutterEmptyTree">
             <English>Declutter Empty Tree</English>
+            <Russian>Очистить пустое дерево</Russian>
             <German>Leer Kategorie aufräumen</German>
             <Polish>Oczyść drzewo pustych obiektów</Polish>
         </Key>
         <Key ID="STR_ZEN_Editor_DeclutterEmptyTree_Description">
             <English>Declutters the empty tree by removing empty vehicles as those are available in the side specific trees</English>
+            <Russian>???</Russian>
```
Comment (mharis001) [inline addons/editor/stringtable.xml]: If you are unsure about the translation, leave empty.

File: addons/attributes/stringtable.xml (hunk 2 above)
Comment (CreepPork) [reply addons/attributes/stringtable.xml]: Line endings probably.

Comment (mharis001) [reply addons/attributes/stringtable.xml]: BOM character, fixed.

**Comment by mharis001** [comment]
Merging to avoid any further conflicts.

**Pull request merged**